src/axolotl/utils/models.py

```diff
@@ -82,8 +82,7 @@ def load_model(
     model_type,
     tokenizer,
     cfg,
-    adapter="lora",
-    inference=False,
```

Comment:
removing this breaks line 99 below since `inference` is no longer defined
`if cfg.device not in ["mps", "cpu"] and inference is False:`

Reply:
Sorry, fixed.